src/ui/public/styles/dark-theme.less

```diff
@@ -313,9 +313,28 @@
 
 // /src/ui/public/vislib/styles/_legend.less
   .legend-col-wrapper {
+    .legend-toggle {
+      background-color: @legend-border;
```

Comment:
Shouldn't this use the @legend-button-bg variable instead of the @legend-border one?

I know they hold the same values, but this is confusing. 

Also, why is it using the @sidebar-hover-color variable in the light theme, and the @legend-border variable in the dark theme?